src/tools/rust-analyzer/crates/ide-db/src/generated/lints.rs

```diff
@@ -6074,6 +6074,17 @@ The tracking issue for this feature is: None.
 
 ------------------------
 
+"##,
+    },
+    Lint {
+        label: "contains",
+        description: r##"# `contains`
+
+The tracking issue for this feature is: [#94047]
+
+[#94047]: https://github.com/rust-lang/rust/issues/94047
+
+
```

Comment:
This file is autogenerated so no need for adding this (its also adding it one slot too late)